packages/carequality-sdk/src/common/util.ts

```diff
@@ -2,5 +2,8 @@ export const XCPD_STRING = "XCPD ITI-55";
 export const XCA_DQ_STRING = "XCA ITI-38";
 export const XCA_DR_STRING = "XCA ITI-39";
 
+export const channelUrls = [XCPD_STRING, XCA_DQ_STRING, XCA_DR_STRING];
```

Comment:
Not a huge deal, but it seems harder to refer to the literal as `XCA_DQ_STRING` than just using its value, `"XCA ITI-38"`. I mean, the compiler tells me the accepted values, and I'd need to know there's a constant somewhere that refers to it.

I'm referring to just declare the type like so:

```ts
export type ChannelUrl = "XCPD ITI-55" | "XCA ITI-38" | "XCA ITI-39";
```

We only need to declare a const array if we need to iterate over the values. And this is easy to adjust later if we need to in the future.